integrations/gradient/src/gradient_haystack/embedders/gradient_document_embedder.py

```diff
@@ -4,9 +4,19 @@
 from gradientai import Gradient
 from haystack import Document, component, default_to_dict
 
+try:
+    from tqdm import tqdm  # type: ignore # noqa
```

Comment:
@masci Requiring some help here - the CI is failing on this line - not sure why.